### DISCUSSION TITLE
M1 clone代码 npm i 后报错

### DISCUSSION THREAD

**Discussion by tipsxBase** [electerm/electerm#3463] [General]
M1 
克隆代码后，执行 npm i，报下面的错误
![image](https://github.com/electerm/electerm/assets/16009561/a1456f77-d97f-4c19-b877-536f8c8aa05d)

**Comment by tipsxBase** [comment]
node 版本 v18.19.0

**Comment by zxdong262** [comment]
可能缺少某些必要组件，试试`xcode-select --install`, `npm i -g node-gyp`

**Reply by tipsxBase** [reply]
这些都已经试过了

**Comment by zxdong262** [comment]
brew install python@3.9
brew install pkg-config
brew install libtool
brew install autoconf
brew install automake

试试

**Comment by TommyCat008** [comment]
<img width="1441" alt="image" src="https://github.com/user-attachments/assets/538ae73b-78be-425a-b98f-94d0a2dbbf9b">
图片这一行注释掉就行了
